backend/api/locationsInsidePrisonApi.test.ts

```diff
@@ -60,4 +60,62 @@ describe('Locations Inside Prison Api', () => {
       ).rejects.toThrow('Bad Request')
     })
   })
+
```

Comment:
Lines 64 to 92 relate to a test missed during previous code changes so added now